dali/test/python/test_eager_cpu_only.py

```diff
@@ -14,6 +14,7 @@
 
 import numpy as np
```

Comment:
A general remark: why is this test separate from the regular test_dali_cpu_only? This seems like a lot of duplication.

Reply:
How would you propose to reuse that code, without some python shenanigans?